server/src/main/java/org/elasticsearch/action/admin/indices/rollover/MetadataRolloverService.java

```diff
@@ -307,6 +309,13 @@ private RolloverResult rolloverDataStream(
             )
             .build();
 
+        newState = IndexWriteLoadForecast.maybeIncludeWriteLoadForecast(
+            dataStream,
+            newState,
+            TimeValue.timeValueDays(7),
+            TimeValue.timeValueHours(8)
```

Comment:
This seems high and I think it excludes all indices that rollover more frequently than every 8 hours from ever having a write load forecast?

I think we can lower it to say 1 hour. But I also wonder if we should also use the avg up time of all candidate shards. Such that we would use `min(avgUpTime, fixedMinimumUpTime)` as the actual minimum uptime.